.helm/ecamp3/templates/api_secrets.yaml

```diff
@@ -21,3 +21,4 @@ data:
   {{- else if .Values.mail.dsn }}
   mailer-dsn: {{ .Values.mail.dsn | b64enc | quote }}
   {{- end }}
+  RECAPTCHA_SECRET: {{ .Values.recaptcha.secret | quote }}
```

Comment:
Gibt es einen spezifischen Grund, weshalb das recaptcha secret nicht Base64 encoded wird? Ist das Secret von Haus aus schon encoded?

Reply:
@carlobeltrame Bräucht es nicht in `api_deployment.yaml` noch einen Eintrag unter `env`? Ist für alle anderen secrets so der Fall. Bin etwas confused, dass es trotzdem funktioniert.

Reply:
Funktioniert nicht so wie intended